client/platform/web-girder/views/Admin/AdminRecents.vue

```diff
@@ -0,0 +1,105 @@
+<script lang="ts">
+import {
+  computed,
+  defineComponent, ref, Ref,
+} from '@vue/composition-api';
+import { getRecentDatasets } from 'platform/web-girder/api/admin.service';
+import type { GirderModel } from '@girder/components/src';
+import moment from 'moment';
+
+
+export default defineComponent({
+  name: 'AdminRecents',
+  setup() {
+    const limit = ref(50);
+    const offset = ref(0);
+    const table: Ref<GirderModel[]> = ref([]);
+    const selected: Ref<{Name: string; Description: string}[]> = ref([]);
+    const headers: Ref<{text: string; value: string}[]> = ref([
+      { text: 'Owner', value: 'owner' },
+      { text: 'User Dir', value: 'userDir' },
+      { text: 'Name', value: 'name' },
+      { text: 'Created', value: 'created' },
+      { text: 'Type', value: 'type' },
+      { text: 'Dataset', value: 'dataset' },
+    ]);
+    // First we need to download the CSV from github
```

Comment:
Does this download a CSV? I thought this was hitting the `get dive_datasets` endpoint

Reply:
The endpoint technically downloads/reads the CSV from github, I'll update to clarify the action.